### PR TITLE
Blocitoff 2 user sign up

### PR DESCRIPTION
**Pull request by kenwmak**
@droberts84 I added the sign up page, you can check out my code. I still need to add the tests, but yeah, I should've wrote the tests first next time.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -50,5 +50,6 @@ end
 gem 'bootstrap-sass'
 gem 'faker'
 gem 'devise'
```
Comment (droberts84) [inline Gemfile]: don't forget to remove any extraneous gems

File: spec/helpers/users_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# describe UsersHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe UsersHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (droberts84) [inline spec/helpers/users_helper_spec.rb]: Unless you plan on adding a user helper file with methods to test, you can probably delete this

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+
+  def check_users(size)
+    expect( User.all.size ).to eq(size)
+  end
+
+  def create_user
+    User.new(name: 'ken mak', email: 'kenwmak@gmail.com', password: 'abc', password_confirmation: 'abc')
```
Comment (droberts84) [inline spec/models/user_spec.rb]: Why not create a for this using Factory Girl?

File: spec/models/user_spec.rb [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  end
+
+  describe "validation" do
+
+    it "validates presence of name" do
+      check_users(0)
+      user = create_user
+      user.name = nil
+      user.save
+      check_users(0)
```
Comment (droberts84) [inline spec/models/user_spec.rb]: Some other ways to test this are to call the `valid? ` method or use the RSpec matcher  `be_valid`

**Comment by droberts84** [comment]
:+1:

**Comment by kenwmak** [comment]
Added the changes to this branch https://github.com/kenwmak/blocitoff/commit/0d0c3a17e03a76e59a045dcb9e9a0d651d37aab5

**Comment by droberts84** [comment]
:+1:

**Pull request merged**